### DISCUSSION TITLE
Improve chat hover functionality

### DISCUSSION THREAD

**Discussion by PetKCH** [Zrips/CMI#5208] [Feature Requests]
Instead of having universal chat hover functionality for all the GroupFormat located in Locale file, implement the ability to have different publicHover for every GroupFormat defined in the config file instead. This would add another level of functionality that I am missing with plugins like DeluxeChat not working properly with 1.16.5 and CMI could easily support! :)

Example in config.yml:

> GroupFormat:
>     '1': '%luckperms_prefix%&f{displayName}{#B7CEEC} » &r{message}'
>     '2': '%luckperms_prefix%&f{displayName}{#B7CEEC} » &r{message}'
>     '3': '%luckperms_prefix%&f{displayName}{#B7CEEC} » &r{message}'
> 
>   GroupMessageFormat:
>     '1': '{message}'
>     '2': '{message}'
>     '3': '{message}'
> 
>   GroupPublicHover:
>     '1': '&eSent time: &6%server_time_hh:mm:ss%'
>     '2': ''&eSent time: &6%server_time_hh:mm:ss% \n&eRank: &7%luckperms_highest_group_by_weight%'
>     '3': ''&eSent time: &6%server_time_hh:mm:ss% \n&eRank: &7%luckperms_highest_group_by_weight%'

**Comment by AZFire1** [comment]
Feature requests belong here https://github.com/Zrips/CMI/discussions

**Comment by CloudeLecaw** [comment]
AZFire is right about this.

but i like his idea:
different hovers for differentgroups when hovering over the chatmeassage. a good idea :)
